src/migrations/20170615084350-create-previously-listened.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
I placed tabs, editor changes it automatically to spaces (BlackChaos knows about this problem)